QuizApp/Quiz/QuizListCollectionViewCell.swift

```diff
@@ -0,0 +1,109 @@
+import Foundation
+import UIKit
+import Kingfisher
+import SnapKit
+
+class QuizListCollectionViewCell: UICollectionViewCell {
+
+    enum Constants {
+
+        static let margins = 10
+        static let trailingConstant = 20
+    }
+
+    static let reuseIdentifier = String(describing: QuizListCollectionViewCell.self)
+
+    private var imageView: UIImageView!
+    private var titleLabel: UILabel!
+    private var descriptionLabel: UILabel!
+    private var difficultyView: DifficultyStackView!
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        buildViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func set(for quiz: QuizModel) {
+        titleLabel.text = quiz.name
+        descriptionLabel.text = quiz.description
+
+        difficultyView.type = DifficultyEnum(rawValue: quiz.difficulty)!
```

Comment:
Avoid force unwrapping, and use a default value instead.
```suggestion
        difficultyView.type = DifficultyEnum(rawValue: quiz.difficulty) ?? .easy
```
Or maybe even better, don't even show the difficulty view if it is `nil`.